devito/parameters.py

```diff
@@ -45,3 +47,4 @@ def _updated(self, key, value):
 parameters = Parameters()
 # Default log level set to INFO
 parameters["log_level"] = 'INFO'
+parameters["autotuning"] = os.environ.get('DEVITO_ATMODE', 'basic')
```

Comment:
I was envisioning this as the place to have defaults only. To be followed up with a function that fills up values from env vars and a corresponding function to do the same from a config file. 

If you prefer to do it this way, could you also do it for the line above (log level)?